### DISCUSSION TITLE
How to mount a block type device permanently even if nodes get rebooted

### DISCUSSION THREAD

**Discussion by liricky** [longhorn/longhorn#10007] [Q&A]
I'm a new user of Longhorn, and I'm very delighted by its v2 engine feature. So I prefer to use v2 engine on all my disks.
Longhorn v2 allows us use block device’s path in the Path field, but the disk block path in "/dev" may change after reboot. (e.g., /dev/sda is an SSD and /dev/sdb is an HDD, after the node reboot, SSD may become /dev/sdb and /dev/sda refers to HDD)
In addition, longhorn doesn't support Symbolic link, and the bus info of some disks may like "scsi@0:2.0.0" by "lshw -c disk -businfo".
So is there any suggestions on how to use such disks with v2 engine, or how to fill the "Path" field. 
Many thx for any help XD

**Comment by derekbit** [comment]
If your disks are NVMe SSD, you can use `nvme` driver that relies on the PCIe address https://longhorn.io/docs/1.7.2/v2-data-engine/features/node-disk-support/#using-nvme-disks. The address shouldn't change after reboot.
